content/en/blog/_posts/2020-07-27-kubernetes-1-17-release-interview.md

```diff
@@ -0,0 +1,219 @@
+
+And I think that is the main skill that translates into programming-- recognizing patterns, finding the things that work, finding the things that don't work. And for me, especially as a collaborative pianist, it's the communicating with people, the finding out what people really want, where something is going, how to figure out what the general direction is that we want to take, before we start writing the first line of code.
+
+**CRAIG BOX: In your experience at Ada or with other experiences you've had, have you been able to identify patterns in other backgrounds for people that you'd recommend, 'hey, you're good at music, so therefore you might want to consider doing something like a course in computer science'?**
+
+GUINEVERE SAENGER: Overall, I think ultimately writing code is just giving a set of instructions to a computer. And we do that in daily life all the time. We give instructions to our kids, we give instructions to our students. We do math, we write textbooks. We give instructions to a room full of people when you're in court as a lawyer.
+
+Actually, the entrance exam to Ada Developers Academy used to have questions from the [LSAT](https://en.wikipedia.org/wiki/Law_School_Admission_Test) on it to see if you were qualified to join the program. They changed that when I applied, but I think that's a thing that happened at one point. So, overall, I think software engineering is a much more varied field than we give it credit for, and that there are so many ways in which you can apply your so-called other skills and bring them under the umbrella of software engineering.
+
+**CRAIG BOX: I do think that programming is effectively half art and half science. There's creativity to be applied. There is perhaps one way to solve a problem most efficiently. But there are many different ways that you can choose to express how you compiled something down to that way.**
+
+GUINEVERE SAENGER: Yeah, I mean, that's definitely true. I think one way that you could probably prove that is that if you write code at work and you're working on something with other people, you can probably tell which one of your co-workers wrote which package, just by the way it's written, or how it is documented, or how it is styled, or any of those things. I really do think that the human character shines through.
+
+**ADAM GLICK: What got you interested in Kubernetes and open source?**
+
+GUINEVERE SAENGER: The honest answer is absolutely nothing. Going back to my programming school— and remember that I had to do a five-month internship as part of my training— the way that the internship works is that sponsor companies for the program get interns in according to how much they sponsored a specific cohort of students.
```

Comment:
I'd write this as:
```suggestion
GUINEVERE SAENGER: The honest answer is absolutely nothing. Going back to my programming school—and remember that I had to do a five-month internship as part of my training—the way that the internship works is that sponsor companies for the program get interns in according to how much they sponsored a specific cohort of students.
```
but that might be European vs. US American punctuation conventions.

Reply:
(GitHub ate my comments, so I'll have a go at remembering them)

We get our transcripts in this format (in ASCII, so with two dashes instead of an em-dash) and I prefer it to indicate a break in speech, as opposed to two linked thoughts.  It's also consistent with previous posts. I'll leave it, but I've s/--/—/ below where I missed them.